libnopegl/src/rtt.c

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright 2023 Matthieu Bouron <matthieu.bouron@gmail.com>
+ * Copyright 2023 Nope Foundry
```

Comment:
this is derived work from `node_rtt.c` so you need to preserve "Copyright 2016-2022 GoPro Inc."

Reply:
Fixed.